scripts/EN/Harry

```diff
@@ -2151,19 +2151,19 @@ CleanScreen(); // 0x270a: 00000064
 LoadBackground(0x2a); // 0x270e: 00001069
 DrawBackground(); // 0x2714: 00000065
 PlayMusic(0x69, 0x1); // 0x2718: 0000206c
-ShowText((1135)"I walk down the hallway. The sleety mixture of snow pounds hard against the windows facing the courtyard. I take a look through a window down at the dark courtyard."); // 0x2720: 0000100c
+ShowText((1135)"I walk down the hallway.¥nW2The sleety mixture of snow pounds hard against the windows facing the courtyard.¥nW2I take a look through a window down at the dark courtyard."); // 0x2720: 0000100c
 HandleInput(); // 0x2726: 00000010
-ShowText((1136)"The not so large courtyard is dotted with areas in which withered plants are growing. In addition to that an old clock tower is faintly visible. From here I cannot determine what time the clock is pointing to."); // 0x272a: 0000100c
+ShowText((1136)"The not so large courtyard is dotted with areas in which withered plants are growing.¥nW1In addition to that an old clock tower is faintly visible.¥nW2From here I cannot determine what time the clock is pointing to."); // 0x272a: 0000100c
 HandleInput(); // 0x2730: 00000010
 OP_0x29(); // 0x2734: 00000029
-ShowText((1137)"Wait. Something appears to be wriggling about in the darkness. An animal? No, it's dark and I can't see so well, but it is slowly moving on two legs. It appears to be searching for prey."); // 0x2738: 0000100c
+ShowText((1137)"Wait.¥nW4Something appears to be wriggling about in the darkness.¥nW2An animal? W2No, W1it's dark and I can't see so well, W1but it is slowly moving on two legs.¥nW2It appears to be searching for prey."); // 0x2738: 0000100c
 HandleInput(); // 0x273e: 00000010
 PlaySoundFx(0xd); // 0x2742: 00001072
-ShowText((1138)"(JiJiJiJi...JiJiJi.........) At that moment the radio I had clipped to my chest suddenly began making noise. As I return my sight to the hallway a sharp pain runs through my thigh."); // 0x2748: 0000100c
+ShowText((1138)"(Hissss…W2Hisss………)¥nAt that moment W1the radio I had clipped to my chest suddenly began making noise.¥nW2As I return my sight to the hallway a sharp pain runs through my thigh."); // 0x2748: 0000100c
```

Comment:
Radio static onomatopoeia.

Used Hiss for English, with additional s letters to match the longer Japanese bits.